src/components/App.tsx

```diff
@@ -6166,6 +6166,13 @@ class App extends React.Component<AppProps, AppState> {
         return;
       }
 
+      if (width === 0 || height === 0) {
+        if (cb) {
+          cb();
+        }
```

Comment:
do we need to call the callback for zero dimensions ? How about instead avoiding calling `updateDOMRect` for zero dimensions ?